modules/test/test_weather.py

```diff
@@ -28,15 +28,15 @@ def validate(actual_name, actual_lat, actual_lon):
             ('27959', check_places("Dare County", "North Carolina")),
             ('48067', check_places("Royal Oak", "Michigan")),
             ('23606', check_places("Newport News", "Virginia")),
-            ('23113', check_places("Midlothian", "Virginia")),
+            ('23113', check_places("Nodstown", "Munster")),
```

Comment:
Whoops, I think I was playing around with tests trying to get them to work and forgot to come back to this one. I'll remove it.